collections/hub/golden_path/feature_tests/transfer_negative_scenarios/fulfil-reserved-v1.0.json

```diff
@@ -561,8 +565,69 @@
               "exec": [
                 ""
               ]
+            },
+            "postRequest": {
+              "exec": [
+                "environment.transferResponse = null",
+                "environment.transferCallback = null",
+                "const expectedTransferState = 'ABORTED'",
+                "",
+                "// lets get retry config",
+                "const maxRetryCount = Number.parseFloat(environment.RETRY_MAX_ATTEMPTS) || 20",
+                "const maxRetryWait = Number.parseFloat(environment.RETRY_MAX_WAIT_IN_MS) || 250",
+                "",
+                "const callbackName = 'put-transfer'",
+                "const callbackPath = `/transfers/{$prev.3.request.body.transferId}`",
+                "",
+                "async function main() {",
+                "  let _response = response",
+                "  let _callback = callback",
+                "",
+                "  for (let retryCount = 0 ; retryCount < maxRetryCount; retryCount++) {",
+                "    console.log(`retry-count=${retryCount+1}, max-count=${maxRetryCount}, max-wait=${maxRetryWait}`)",
+                "    ",
+                "    const isOk = _callback?.body?.transferState === expectedTransferState;",
+                "    ",
+                "    if (isOk) {",
+                "      console.log(`Everything is OK, let's continue`)",
+                "      environment.transferResponse = _response",
+                "      environment.transferCallback = _callback",
```

Comment:
If you don't need to access these variables in following requests in the test case, you can use requestVariables also